R/conditions.R

```diff
@@ -191,7 +248,22 @@ createStackTracePromiseDomain <- function() {
 }
 
 deepStacksEnabled <- function() {
-  getOption("shiny.deepstacktrace", TRUE)
+  deepStackLimit() > 0L
+}
+
+deepStackLimit <- function() {
+  opt <- getOption("shiny.deepstacktrace", 8L)
+  if (!is.numeric(opt) && !is.logical(opt)) {
+    opt <- FALSE
+  }
+
+  if (isFALSE(opt)) {
```

Comment:
`isFALSE()` was added in R 3.4.2, and we currently depend on R 3.0.2. Lets do `rlang::is_false()` or `identical(opt, FALSE)`?

Reply:
bslib uses `isFALSE()` (and has for almost 2 years now), maybe we should adjust the minimum R version in both places?